md/release_notes.md

```diff
@@ -14,16 +31,11 @@
 
 ## Bug fixes
 
-### Fixes in Bonita Intelligent Continuous Improvement 1.0.2
-* ICI-854  As Omar, I don't want to see deleted cases (either automatic or manual delete) in my case list
-* ICI-901  Installer should not display password in debug mode
-* ICI-900  Typo in interactive installer 
-* ICI-899  Start command in evaluation mode should not ask host name from bonita
-
-### Fixes in Bonita Intelligent Continuous Improvement 1.0.1
-* ICI-848 Document why deleted cases are displayed in Operations Management Living Application	
-* ICI-826 Some logs are printed when connection to database is not UP	
-* ICI-837 Polling status displays "finishing" status before "running"	
-* ICI-847 No percentage displayed when case is late since less than 1 hour
-* ICI-853 Connection errors to BICI storage are not displayed properly in polling status page
-* ICI-855 Polling fails when a case is deleted in Bonita database
+### Fixes in Bonita Intelligent Continuous Improvement 1.1.0
+ICI-1045 On configuration page, unable to update "Confidence threshold" if "Target duration" is empty
```

Comment:
layout is broken ( `*`are missing)  . see 

![image](https://user-images.githubusercontent.com/6574899/48251800-3204a480-e403-11e8-9118-dda072c657ce.png)